src/components/color-picker/utils.ts

```diff
@@ -0,0 +1,217 @@
+import tinycolor, { type ColorInput } from 'tinycolor2';
+
+export type ColorFormat = 'rgb' | 'hex';
+
+export function roundTwoDecimal(num: number) {
+  return Math.round(num * 100) / 100;
+}
+
+export class Color {
+  instance: tinycolor.Instance;
```

Comment:
small nitpick here if we are not changing the instance from outside the class, which we should not be doing anyway, can we mark the instance as readonly?

Reply:
None blocking we can adjust at any other PR.